docs/guides/index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: Guides
+description: Achieve more with lakeFS by reading these practical guides
+nav_order: 42
+has_children: true
+--- 
+
+# Guides
+
+Explore these practical how-to guides to make the most of lakeFS. 
+
+## Monitoring with Prometheus
+
+This guide shows which metrics lakeFS exposes, a selection of example queries, and example of a Grafana dashboard, and how to make it all work.
+
+[Learn more](https://docs.lakefs.io/reference/monitor.html)
+
+## Authentication and authorization
+
+lakeFS authenticates users from a built-in authentication database, or optionally from a configured LDAP server. Explore how authentication and authorization work in lakeFS.
+
+[Learn more](https://docs.lakefs.io/reference/authentication.html)
+
+## Upgrading lakeFS
+
+Upgrading lakeFS usually requires re-deploying with the latest image or downloading the latest version, if you’re using the binary.
+
+[Learn more](https://docs.lakefs.io/reference/upgrade.html)
```

Comment:
I don't understand this either.  I think this PR **renames** this file and will break this link.